eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/ClusteringWriter.java

```diff
@@ -194,17 +213,8 @@ private void processClusteredRemappedVariants(List<? extends SubmittedVariantEnt
 
         String assembly = clusteredRemappedSubmittedVariants.get(0).getReferenceSequenceAccession();
```

Comment:
Do you need to get the assembly from the clustered variant now that you have it populated in the constructor ?

Reply:
Actually now that the assembly is a property you can avoid passing it around a lot of methods